src/HealthChecks.ApplicationStatus/HealthChecks.ApplicationStatus.csproj

```diff
@@ -8,7 +8,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Extensions.Diagnostics.HealthChecks" Version="8.0.0" />
+    <PackageReference Include="Microsoft.Extensions.Diagnostics.HealthChecks" Version="8.0.6" />
```

Comment:
It seems that all projects reference it, so we should most likely move this reference to the `Directory.Build.props` file here:

https://github.com/Xabaril/AspNetCore.Diagnostics.HealthChecks/blob/42c2c732a5369849d59f9d79cdd4be497da98786/Directory.Build.props#L17-L18

So next time we update only one file.